docs/eventing/samples/container-source/README.md

```diff
@@ -0,0 +1,133 @@
+ContainerSource will start a container image which will generate events under contain situations and send messages to a sink uri. ContainerSource could be an easy way to implement your own event sources. This document shows how to configure ContainerSource as event source for functions and gives instructions on how to create an event source using ContainerSource.
+
+## Create a heartbeats ContainerSource
```

Comment:
Which does what ? and why are we doing this ?

The intro makes it sound that this doc will show us how to write our own source and it starts with an example usage. We should explain that clearly

Reply:
Interesting point.

The heartbeats sample is not only a sample of how to use an existing ContainerSource but also a sample of how to develop a ContainerSource. That's why this document includes both sides.

Reply:
Then I think we should explain the code almost line by line.